src/components/Clicker.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+class Clicker extends React.Component {
+    
+        state = { count: 0 };
```

Comment:
This is not the way we want to structure our state. We want to create state as the main object that hold all of our data.